### PR TITLE
Add Image item in Photo Gallery.

### PR DESCRIPTION
**Pull request by iiLubos**
image
 
```
MMPhotoGallery {
  showAddImage: true
  onAddImage: console.log("Open camera")
  ...
}
```

### CODE REVIEW

File: app/icons/AddImage.svg
Comment (tomasMizera) [inline app/icons/AddImage.svg:1]: Shouldn't this be white icon?

File: app/mmstyle.h [hunk 1]
```diff
@@ -54,7 +54,7 @@ class MMStyle: public QObject
     // Colors - secondary palette
     Q_PROPERTY( QColor lightGreenColor READ lightGreenColor CONSTANT )
     Q_PROPERTY( QColor mediumGreenColor READ mediumGreenColor CONSTANT )
-    Q_PROPERTY( QColor greyColor READ greyColor CONSTANT )
```
Comment (tomasMizera) [inline app/mmstyle.h:57]: Heh, I changed it to `grey` in the previous PR. Looks like a war between US and UK english https://thisonevsthatone.com/grey-vs-gray/ :D

Comment (tomasMizera) [reply app/mmstyle.h:57]: I'd prefer to stick to `grey` though, it is how the design system names it. It makes it easier then to apply it

File: app/qml/components/MMPhotoGallery.qml [hunk 2]
```diff
@@ -21,9 +21,11 @@ Item {
   property string warningMsg
   property string errorMsg
   property int maxVisiblePhotos: 5 // -1 for showing all photos
```
Comment (tomasMizera) [inline app/qml/components/MMPhotoGallery.qml:23]: Can we make `-1` default here please?

File: app/icons/AddImage.svg
Comment (iiLubos) [reply app/icons/AddImage.svg:1]: All icons should be white. We can change all icons in one separate PR later

**Review by tomasMizera — APPROVED** [review]

**Pull request merged**